setupVIP.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -eu
+
+${MULTIPASSCMD}  mount $PWD generic-rke2-master-1:/hostFileDir || true
```

Comment:
Maybe use transfer to avoid mount "issues" with different host operating systems.

Why default to true, when this fails we cannot continue as the following exec relies on the mount being available.

Reply:
Good fedback @superseb 
The `true` was to allow me to rerun the script without stopping because of the `bash -e` and a already mounted drive. I suggest taking out the `|| true` to clarify this. I don't know of a reason why `transfer` is preferable otherwise and in the interests of minimizing storage in the VM (or removing it after the transfer) would leave this as a `mount`.

Reply:
Ok I ran into https://github.com/canonical/multipass/issues/1389 but it's fine as there is an easy solution.